### PR TITLE
Use stable nodejs v22

### PR DESCRIPTION
**Pull request by eygraber**
- The version still has to be specified until the Kotlin tooling updates

### CODE REVIEW

**Comment by eygraber** [comment]
Ah actually looks like this won't be possible until 2.0 RC3 because Kotlin needs to remove the `--experimental-wasm-gc` flag.

**Comment by eygraber** [comment]
The fix was backported to Kotlin 1.9.24. Just need [an updated version of KSP](https://github.com/google/ksp/issues/1874) and this should be mergeable.

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [versions]
 kotlin-inject = "0.7.0-SNAPSHOT"
-kotlin = "1.9.22"
-ksp = "1.9.22-1.0.17"
+kotlin = "1.9.24"
+ksp = "1.9.23-1.0.20"
```
[the remark this answers is not in the document]
Comment (eygraber) [reply gradle/libs.versions.toml:4]: Yes, but that version isn't released yet.

[the remark this answers is not in the document]
Comment (evant) [reply gradle/libs.versions.toml:4]: Oh gotcha

[the remark this answers is not in the document]
Comment (eygraber) [reply gradle/libs.versions.toml]: 1.9.24-1.0.20 was released, and I updated the PR

**Review by evant — APPROVED** [review]

**Pull request merged**